Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.5.0"
 
 [deps]
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Polynomials = "f27b6e38-b328-58d1-80ce-0feddd5e7a45"
```

Comment:
I don't think this is needed here?

Reply:
@jeremylt, which of the packages do you mean is not needed in the ```Project.toml``` files here and in the jupyter folder?

Reply:
Polynomials isn't directly used in any of our examples anymore, right?